package.json

```diff
@@ -72,30 +72,35 @@
     "cross-env": "^7.0.0",
     "http-errors": "^1.7.3",
     "husky": "^4.2.1",
-    "jest": "^25.1.0",
+    "jest": "^26.1.0",
     "jest-junit": "^10.0.0",
     "lint-staged": "^10.0.2",
     "middy-middleware-json-error-handler": "^2.0.0",
-    "pkg-ok": "^2.3.1",
+    "pkg-ok": "^3.0.0",
     "prettier-standard": "^16.0.0",
     "reflect-metadata": "^0.1.13",
     "rimraf": "^3.0.0",
-    "semantic-release": "^17.0.0",
+    "semantic-release": "^16.0.0",
```

Comment:
Thanks for updating! I also see that you downdated `semantic-release` and `tslint`. What's the reason?

Reply:
It was throwing lots of peer dependency issue while doing npm install. I've modified those version to fix this peer dependency issue.

Reply:
Thanks! If this turns out to be complicated, please ping me and I'll take a look and I'll update the dependencies in a separate PR.

Reply:
Can you please fix those issues then? That will be helpfull for next contributor too.